installer/packer/vic-unified.ovf

```diff
@@ -294,6 +294,38 @@ EVALUATION LICENSE. If You are licensing the Software for evaluation purposes, Y
         <Description>Paste in the content of certificate key file. Leave blank for a generated key.</Description>
       </Property>
     </ProductSection>
+    <ProductSection ovf:class="engine_installer" ovf:required="false">
+      <Info>VIC Engine Install Wizard Properties</Info>
+      <Category>6. VIC Engine Install Wizard Configuration</Category>
+      <Property ovf:key="deploy" ovf:type="boolean" ovf:userConfigurable="true" ovf:value="false">
+        <Label>6.1. Enable VIC Engine Install Wizard</Label>
+        <Description>When setting this to true, starts the VIC Engine Install Wizard web application.</Description>
+      </Property>
+      <Property ovf:key="vcenter_addr" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+        <Label>6.2. vCenter Address</Label>
+        <Description>IP address or FQDN of the vCenter. Leave blank if not using VIC Engine Install Wizard.</Description>
+      </Property>
+      <Property ovf:key="admin_user" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
+        <Label>6.3. vCenter Admin Username</Label>
+        <Description>Username of the vCenter admin user. Leave blank if not using VIC Engine Install Wizard.</Description>
+      </Property>
+      <Property ovf:key="admin_password" ovf:password="true" ovf:qualifiers="MinLen(0),MaxLen(65535)" ovf:type="string" ovf:userConfigurable="true" ovf:value="">
```

Comment:
We shouldn't be asking for vcenter credentials here, we should ask them in the engine installer.